packages/xrpl/test/integration/transactions/escrowCancel.test.ts

```diff
@@ -1,72 +1,133 @@
-import { generateFundedWallet, getXRPBalance, testTransaction } from '../utils'
+import {
+  setupClient,
+  teardownClient,
+  type XrplIntegrationTestContext,
+} from '../setup'
+import {
+  // calculateWaitTimeForTransaction,
+  generateFundedWallet,
+  // getXRPBalance,
+  testTransaction,
+  runCommand,
+} from '../utils'
 
 // how long before each test case times out
-const TIMEOUT = 20000
+const TIMEOUT = 50000
 
 describe('EscrowCancel', function () {
-  this.timeout(TIMEOUT)
-
-  beforeEach(_.partial(setupClient, serverUrl))
-  afterEach(teardownClient)
-
-  it('base', async function () {
-    // get the most recent close_time from the standalone container for cancel & finish after.
-    const CLOSE_TIME: number = (
-      await this.client.request({
-        command: 'ledger',
-        ledger_index: 'validated',
-      })
-    ).result.ledger.close_time
-    const wallet1 = await generateFundedWallet(this.client)
-
-    const createTx: EscrowCreate = {
-      Account: this.wallet.classicAddress,
-      TransactionType: 'EscrowCreate',
-      Amount: '10000',
-      Destination: wallet1.classicAddress,
-      CancelAfter: CLOSE_TIME + 3,
-      FinishAfter: CLOSE_TIME + 2,
-    }
-
-    await testTransaction(this.client, createTx, this.wallet)
-
-    const initialBalanceWallet1 = await getXRPBalance(this.client, wallet1)
-
-    // check that the object was actually created
-    const accountObjects = (
-      await this.client.request({
-        command: 'account_objects',
-        account: this.wallet.classicAddress,
-      })
-    ).result.account_objects
-
-    assert.equal(accountObjects.length, 1)
-
-    const sequence = (
-      await this.client.request({
-        command: 'tx',
-        transaction: accountObjects[0].PreviousTxnID,
-      })
-    ).result.Sequence
-
-    const cancelTx: EscrowCancel = {
-      TransactionType: 'EscrowCancel',
-      Account: this.wallet.classicAddress,
-      Owner: this.wallet.classicAddress,
-      OfferSequence: sequence,
-    }
-
-    await testTransaction(this.client, cancelTx, this.wallet)
-
-    assert.equal(
-      await getXRPBalance(this.client, wallet1),
-      initialBalanceWallet1,
-    )
+  let testContext: XrplIntegrationTestContext
+
+  beforeEach(async () => {
+    testContext = await setupClient(serverUrl)
   })
+  afterEach(async () => teardownClient(testContext))
+
+  it(
+    'base',
+    async () => {
+      // Funding the wallet can take some time, so we do it first BEFORE getting the ledger close_time.
+      const wallet1 = await generateFundedWallet(testContext.client)
+
+      // get the most recent close_time from the standalone container for cancel & finish after.
+      const CLOSE_TIME: number = (
+        await testContext.client.request({
+          command: 'ledger',
+          ledger_index: 'validated',
+        })
+      ).result.ledger.close_time
+
+      // const waitTimeInMs = calculateWaitTimeForTransaction(CLOSE_TIME)
```

Comment:
There's a couple spots within this file with commented code, should they just be removed?

Reply:
Or if not can you add a comment explaining when someone would want to uncomment them? (Ex. When debugging XYZ)

Reply:
They are left intentionally and there's a comment about making this test more robust in the future. The comment is further down in the file.

Reply:
Done